### PR TITLE
Improving icloud device tracker

### PR DESCRIPTION
**Pull request by evgeniy-khatko**
Updating documentation for new pull request to main HA codebase: "Improving icloud device tracker"

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14078

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by evgeniy-khatko** [comment]
Anyone can take a look at this?
Thanks!

File: source/_components/device_tracker.icloud.markdown [hunk 1]
```diff
@@ -48,7 +50,7 @@ To disable the drainage of the battery, a dynamic interval is being used for eac
 2 Factor Authentication is the improved version of 2 Steps Authentication, this is still not supported by the pyicloud library. Therefore it's not possible to use it with the device_tracker yet.
 
 4 services are available for this component:
-- **icloud_update**: This service can be used to ask for an update of a certain iDevice. The `account_name` and `device_name` are optional.
+- **icloud_update**: This service can be used to ask for an update of a certain iDevice. The `account_name` and `device_name` are optional. Request will result in new Home Assistant event describing current iphone location. Can be used in automations when manual location update is needed, e.g. to check if anyone is home when door's been opened.
```
Comment (arsaboo) [inline source/_components/device_tracker.icloud.markdown]: Can you indicate what event is fired?

File: source/_components/device_tracker.icloud.markdown [hunk 2]
```diff
@@ -33,9 +33,11 @@ Configuration variables:
 - **username** (*Required*): The username for the iCloud account.
 - **password** (*Required*): The password for your given username.
 - **account_name** (*Optional*): The friendly name for the account_name. If this isn't given, it will use the account_name of the username (so the part before the `@` in the email address).
+- **max_interval** (*Optional*): Maximum interval in minutes between subsequent location checks. This tracker uses dynamic intervals for requesting location updates. When iphone is stationary, interval will eventually be set to `max_interval`. This saves battery. E.g. setting it to 10 means that when you start moving after being stationary, maximum 10 min will pass until your location starts updating. Default is 30 min. Very minimum value is 1 min.
```
Comment (arsaboo) [inline source/_components/device_tracker.icloud.markdown]: Can you simplify what does `For example, setting it to 10 means that when you start moving after being stationary, maximum 10 min will pass until your location starts updating.` mean? I am guessing, if the phone is stationary, it updates every 10 minutes. What about when the phone is not stationary?

File: source/_components/device_tracker.icloud.markdown (hunk 1 above)
Comment (evgeniy-khatko) [reply source/_components/device_tracker.icloud.markdown]: Updated with link to state_changed event.

File: source/_components/device_tracker.icloud.markdown (hunk 2 above)
Comment (evgeniy-khatko) [reply source/_components/device_tracker.icloud.markdown]: Rephrased a bit. Let me know if it still sounds confusing.

**Review by fabaff — APPROVED** [review]
Thanks :bird:

**Pull request merged**